packages/graphql/package.json

```diff
@@ -58,7 +58,7 @@
     "rlp": "^2.2.2",
     "subscriptions-transport-ws": "^0.9.16",
     "web3": "1.0.0-beta.34",
-    "web3-provider-engine": "^15.0.0"
+    "web3-provider-engine": "mikeshultz/web3-provider-engine#too-many-requests"
```

Comment:
I would recommend adding a TODO to replace this package with the official one once https://github.com/MetaMask/web3-provider-engine/pull/315 is accepted and merged in.

Reply:
Would if I could.  Can't do comments in JSON, sadly.  Recommend somewhere else?

Reply:
ah darn... how about in README ? not ideal but can't think of a better place ?

Reply:
`TODO.md`?  Or is that too extreme when we have things like GH issues?

Reply:
Actually you are right. Filing a GH issue if probably better... I'd vote for that.

Reply:
https://github.com/OriginProtocol/origin/issues/2388